public/api/conf/1.0/application.yaml

```diff
@@ -69,7 +69,7 @@ components:
       name: Authorization
       in: header
       required: true
-      description: An [OAuth 2.0 Bearer Token](http://localhost:9680/api-documentation/docs/authorisation/application-restricted-endpoints) with the write:transfer-complete scope.
+      description: An [OAuth 2.0 Bearer Token](http://localhost:9680/api-documentation/docs/authorisation/application-restricted-endpoints) with the read:bulk-data-download-list scope.
```

Comment:
Should the link point to https://developer.service.hmrc.gov.uk/api-documentation/docs/authorisation instead of localhost:9680?